install/windows/docker-ee.md

```diff
@@ -1,23 +1,23 @@
 ---
-description: How to install Docker EE for Windows Server
-keywords: Windows, Windows Server, install, download, ucp, Docker EE
-title: Install Docker Enterprise Edition for Windows Server
+description: How to install Docker Engine - Enterprise for Windows Server
+keywords: Windows, Windows Server, install, download, ucp, Docker Engine - Enterprise
+title: Install Docker Engine - Enterprise on Windows Servers
 redirect_from:
 - /docker-ee-for-windows/install/
 - /engine/installation/windows/docker-ee/
 ---
 
 {% capture filename %}{{ page.win_latest_build }}.zip{% endcapture %} {% capture download_url %}https://download.docker.com/components/engine/windows-server/{{ site.docker_ee_version }}/{{ filename }}{% endcapture %}
 
-Docker Enterprise Edition for Windows Server (*Docker EE*) enables native Docker containers on Windows Server. Windows Server 2016 and later versions are supported. The Docker EE installation package includes everything you need to run Docker on Windows Server.  This topic describes pre-install considerations, and how to download and install Docker EE.
+Docker Engine - Enterprise enables native Docker containers on Windows Server. Windows Server 2016 and later versions are supported. The Docker Engine - Enterprise installation package includes everything you need to run Docker on Windows Server.  This topic describes pre-install considerations, and how to download and install Docker Engine - Enterprise.
 
 > Release notes
 >
 > [Release notes for all versions](/release-notes/)
 
-## Install Docker EE
+## Install Docker Engine - Enterprise
 
-Docker EE for Windows requires Windows Server 2016 or later. See
+Docker Engine - Enterprise requires Windows Server 2016 or later. See
```

Comment:
This link is not helpful at all in the install process - it needs to be a system requirements page. As a customer I would expect that link to actually tell me about pre-install configuration and system requirements.
This one gives no useable information, it's a link to other links and it's in a confusing location in the doc near Next steps  https://docs.docker.com/install/windows/docker-ee/#what-to-know-before-you-install